addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -915,9 +915,10 @@ var SnippetEditor = Widget.extend({
      */
     _outPreviousDropzone(self, currentDropzoneEl) {
         const previousDropzoneEl = this;
-        const rowEl = previousDropzoneEl.parentNode;
+        const parentEl = previousDropzoneEl.parentNode;
```

Comment:
As much as I really don't think changing the name of this one particular variable, which is not added to any extensible object, would cause an issue to anyone, I still think cosmetic changes that do no provide any sort of fix should only be done in master, just for the sake of respecting the rules written [here](https://github.com/odoo/odoo/wiki/Reviewing#targeting-stable). 

It actually creates a lot of noise on the commit, and removes history, on the lines. So I guess that's another argument for not changing the name 😉

Reply:
> Re-reading myself, I think it is better for master indeed 🤔

Then what are we disagreeing on? 😉 

>  I think it is to make the code less confusing 

This is my opinion and it could be wrong, but code readability is cosmetic, since we're not fixing a bug, nor introducing or correcting a feature.

> it is more a variable correction than a variable renaming

Not sure what you mean by that, don't you just change the name of the variable?

Reply:
Okay I will remove that and add it back in master (in its own commit) :slightly_smiling_face: 

> Not sure what you mean by that, don't you just change the name of the variable?

It looks like that but what I meant is that calling it `rowEl` in not correct, since depending on the block we are dragging, it may not be a row element at all, so the correction fixes this confusion. But I agree with you that it does not fix a real bug, and that it therefore is a cosmetic change :wink: